### ISSUE TITLE
Enable easy copying of the username.

### ISSUE THREAD

**Issue by jim4067** [coral-xyz/backpack#2094]
Enable a user to copy his/her/their username if clicked on. Sort of like how it happens on Discord.

**Comment by Merdi-kim** [comment]
Yeah, this could be an enhancement. I don't know if @jim4067 is part of the backpack team or not because I'd like to get an approval from the team before moving forward with this feature

**Comment by jim4067** [comment]
Hello @Merdi-kim 👋, 

I'm not part of the team, but you can go for it. I don't think there'd be a problem, @hkirat, right?

**Comment by Merdi-kim** [comment]
I'll be waiting for a go-ahead from someone from the team. But overall I think this could be a nice feature

**Comment by thehashmap** [comment]
Can I work on this feature?
